app/src/main/java/de/danoeh/antennapod/fragment/PlaybackHistoryFragment.java

```diff
@@ -71,15 +87,68 @@ public void onConfirmButtonPressed(DialogInterface dialog) {
             };
             conDialog.createNewDialog().show();
 
+            return true;
+        } else if (item.getItemId() == R.id.playback_remove_filter) {
```

Comment:
Instead of having a second menu item to remove the filter, is it possible to add a "clear" button to the date picker? Then this reduces the number of buttons when not using the feature.

Reply:
I agree that this would be the preferable solution. However, after quite some researching on how to do this, I found that there is currently an open [PR](https://github.com/material-components/material-components-android/pull/2371) that will implement exactly this feature. So, I suggest that we wait until this PR gets merged. So the current solution will be a "placeholder" until they release the official "clear" button.